src/components/Card.js

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components';
+
+export default function Card({ activity, friend, emptytext }) {
+  return (
+    <Wrapper>
+      <p>
+        <strong>{activity}</strong>
+      </p>
+      <p>{friend}</p>
+      {emptytext}
+    </Wrapper>
+  );
+}
+
+const Wrapper = styled.section`
```

Comment:
You already have a Wrapper in your App.js - it might not be important, but consider naming them differently (i.e. Wrapper--App & Wrapper--Card or something like this).

Reply:
Good idea, thank you!😊